corehq/apps/export/models/new.py

```diff
@@ -3131,12 +3131,20 @@ def get_export_column(column):
             selected=True,
         )
 
+    max_sheet_name_length = 31
```

Comment:
nit: how about making this a constant?

`This is an issue for all exports with long table id's or sheet names`
Additionally, if this is an issue with all exports, it would be nice to extract this logic into a util function which other exports can use as well.

Reply:
> ... it would be nice to extract this logic into a util function which other exports can use as well.

The reason I didn't want to do this is to prevent confusion, because for other exports the user specifies the sheet name (if not, we default to something like "case" / "from", so the defaults wouldn't be an issue). If the user specified a sheet name, but sees a "{domain}_{random_id}" (if even possible), that will be confusing. I think we might be better off preventing users from entering long names?

Reply:
https://github.com/dimagi/commcare-hq/pull/33490/commits/4411afa1928b4c0b6ebc340f29cff327333eac58 cc @Charl1996

Reply:
> other exports the user specifies the sheet name 

Ya, I meant only if the name isn't specified. It was a non blocking feedback, so cool to skip.